python/doc/examples/probabilistic_modeling/distributions/plot_pointConditionalDistribution.py

```diff
@@ -0,0 +1,255 @@
+"""
+Create a Point Conditional Distribution
+=======================================
```

Comment:
the file is duplicated:
git rm python/doc/examples/probabilistic_modeling/distributions/plot_pointConditionalDistribution.py

Reply:
When I execute the command, I obtain: 

git rm python/doc/examples/probabilistic_modeling/distributions/plot_pointConditionalDistribution.py
fatal : le chemin 'python/doc/examples/probabilistic_modeling/distributions/plot_pointConditionalDistribution.py' ne correspond à aucun fichier

Reply:
yes its already moved